pom.xml

```diff
@@ -37,12 +37,11 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <raml-module-builder-version>33.2.7</raml-module-builder-version>
+    <raml-module-builder-version>34.0.0</raml-module-builder-version>
```

Comment:
Why not 34.1.0?

Reply:
34.0.* is the official version that will receicve security patches for Morning Glory.
34.1.* isn't.